CONTRIBUTING.md

```diff
@@ -1,12 +1,40 @@
 # Contributing Guidelines
-Feel free to contribute at any time to this project. 
-To contribute, 
-- Create an issue with a bug description or feature request
-- OR fix the issue / add a feature yourself by creating a pull request
 
-# How to run?
-Download/clone the repo and run 
+Thank you for considering contributing to this project! We welcome any contributions, big or small. Here are some guidelines to help you get started:
+
+## Reporting Issues
+
+If you encounter a bug or have a feature request, please create an issue in the GitHub repository. Be sure to provide a clear and detailed description of the problem, including any steps needed to reproduce it. 
+
+## Making Changes
+
+If you want to fix an issue or add a new feature, we encourage you to create a pull request (PR). Here are the steps to follow:
```

Comment:
I would probably add something like this. 
"If you are not sure whether your feature is desired, feel free to ask in an issue first. "

Reply:
Sure, that sounds good